### PR TITLE
[WIP] 14454-appeals-tab-aria

### PR DESCRIPTION
**Pull request by 1Copenut**
## Description

The Track Claims and Appeals tabs for the Appeals section were missing some of the controls present on the Claims tabs, and this was causing both automated errors, and manual usability issues for screen readers.

Related Issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14454

## Testing done
* Tested locally with axe-core to ensure there were no markup or automated error regressions
* Tested locally with MacOS + Safari + VoiceOver
* Will continue testing in Staging for JAWS and NVDA (can't test these locally because I can't log in a separate machine from the one running the website and API)

## Acceptance criteria
- [ ] Automated errors are no longer appearing for `aria-controls` attribute referencing a non-present ID

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by cvalarida — COMMENTED** [review]
I don't happen to have access to a proper screen reader combination these days; can you screenshot or record or type out what the screen reader says?

I think the code looks good, but I can't really test it out myself.

File: src/applications/claims-status/containers/AppealsV2DetailPage.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { addStatusToIssues } from '../utils/appeals-v2-helpers';
 const AppealsV2DetailPage = ({ appeal }) => {
   const issues = addStatusToIssues(appeal.attributes.issues);
   return (
-    <div>
+    <div aria-labelledby="tabv2detail" id="tabPanelv2detail" role="tabpanel">
```
[the remark this answers is not in the document]
Comment (1Copenut) [reply src/applications/claims-status/containers/AppealsV2DetailPage.jsx]: `aria-labelledby` reads out the tab name when the `tabPanel` is under the assistive device's virtual cursor. In the case of VoiceOver, it's the difference in the entire tab panel being announced as "Status, tab panel". Big gain for accessibility.

[the remark this answers is not in the document]
Comment (cvalarida) [reply src/applications/claims-status/containers/AppealsV2DetailPage.jsx]: Ahhh, very cool. Thanks!  :raised_hands:

**Review by cvalarida — APPROVED** [review]

File: src/applications/claims-status/components/TabItem.jsx [hunk 2]
```diff
@@ -14,24 +14,35 @@ class TabItem extends React.Component {
     document.removeEventListener('keydown', this.tabShortcut);
   }
 
+  // Grab the current URL, trim the leading '/', and return activeTabPath
+  trimCurrentUrl() {
```
Comment (markgreenburg) [inline src/applications/claims-status/components/TabItem.jsx]: Does `trimCurrentUrl` need to be bound to `this`?

**Review by markgreenburg — COMMENTED** [review]
Not sure if this PR is still a thing (sorry it took so long to get to), but looks good, just a question on binding one of the methods.

File: src/applications/claims-status/components/TabItem.jsx (hunk 2 above)
Comment (1Copenut) [reply src/applications/claims-status/components/TabItem.jsx]: @markgreenburg That's a good call, I do believe `trimCurrentUrl` does need to be bound. I'll refactor and retest before merging. Thank you.

Comment (1Copenut) [reply src/applications/claims-status/components/TabItem.jsx]: @markgreenburg I updated this as `trimCurrentUrl = () => { ... }`. Passes the tests, and worked locally.

**Review by markgreenburg — APPROVED** [review]
🚀

**Pull request merged**